apps/backend/src/modules/quiz-questions/quiz-question.service.ts

```diff
@@ -89,6 +94,38 @@ class QuizQuestionService implements Service {
 		return { items: result };
 	}
 
+	public async findQuestions(
+		query: CategoriesGetRequestQueryDto,
```

Comment:
I'm not sure, but I think it is better to destructure this object and then use only `categoryIds`